lib/tasks/update_application_form_status.rake

```diff
@@ -0,0 +1,10 @@
+namespace :db do
+  desc "Update application status"
+  task update_status: :environment do
+    # fetch all application forms with 'initial assessment' status
+    forms = ApplicationForm.where(status: "initial_assessment")
+
+    # update the status for each form
+    forms.each { |form| form.update(status: "assessment_in_progress") }
```

Comment:
This would work, but it would be quite slow as it would require a query to do the database for each application form. If we do something like `forms.update_all(state: "assessment_in_progress")` that'll do the update in one single query and let the database handle that for us.